cssResponsive/css-responsive_responsive-website/css/styles.css

```diff
@@ -243,11 +244,32 @@ CSS when certain conditions that we declare are met - such as a certain viewport
 width, etc. */
 
 /* Media Query Tablet */
-/* --- */
-/* Replace this comment with your media query. */
-/* --- */
-
-/* Media Query Desktop */
-/* --- */
-/* Replace this comment with your media query. */
-/* --- */
+@media screen and (min-width: 600px) {
+  .card-section {
+    display: flex;
+    justify-content: center;
+    flex-flow: wrap;
+    gap: var(--gap);
+  }
+
+  .product-card {
+    flex-wrap: wrap;
+    width: 40%;
+  }
+  .intro-text {
+    align-self: start;
+    width: 40%;
+  }
+}
+
+@media screen and (min-width: 900px) {
+  .card-section {
+    display: flex;
+    justify-content: center;
+    flex-flow: wrap;
```

Comment:
What's the difference between `flex-wrap` and `flex-flow`?

Reply:
so i dont need to do flex-direction but maybe i didnt need it in the first place. i will check it

Reply:
yeah so it doesnt work with flex-wrap: wrap; only
so flex-flow: wrap; makes more sence